sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_base_client.py

```diff
@@ -26,8 +26,8 @@ def _authentication_policy(credential):
 
 class TextAnalyticsClientBase(object):
     def __init__(self, endpoint, credential, **kwargs):
-        api_version = kwargs.pop("api_version", None)
-        _TextAnalyticsClient = load_generated_api(api_version)
+        self._api_version = kwargs.pop("api_version", ApiVersion.V3_0)
```

Comment:
`ApiVersion.V3_0` or `ApiVersion.V3_1_Preview_1`?

Reply:
I'm going to make a separate PR to default to v3.1-preview.1. For this PR, we had issues with `detect_langauge` (thanks for looking into that!) and it would require re-recording all of our tests, so wanted to address it in a separate PR